wsidicom/instance/dataset.py

```diff
@@ -238,6 +238,18 @@ def frame_offset(self) -> int:
                 "Concatenated file missing concatenation frame offset" "number"
             )
 
+    @cached_property
+    def tile_frame_count(self) -> int:
+        """Return number of frames corresponding to tiles in instance."""
+        frame_count = self.frame_count
+
+        if self.image_type == ImageType.VOLUME:
```

Comment:
As far as I know, there is nothing in the standard that prevents labels and overviews from having multiple focal planes and/or optical paths.

Reply:
[This example](https://imagingdatacommons.github.io/slim/studies/2.25.23897195960526781231486877255455994829/series/2.25.83282858720704132758110891374375550907) that I referenced earlier contains a label that reports `NumberOfFrames` as `1` and `NumberOfOpticalPaths` as `3`. So the number of frames apparently doesn't include the number of optical paths for that label. I'm not sure if that's because it is a label, or if that should be considered an invalid DICOM dataset.

Removing that line caused errors for me loading that series, so we might need to figure out an alternative if that won't work.